src/Microsoft.AspNetCore.Razor.Evolution/RazorEngine.cs

```diff
@@ -16,10 +16,24 @@ public static RazorEngine Create()
         public static RazorEngine Create(Action<IRazorEngineBuilder> configure)
         {
             var builder = new DefaultRazorEngineBuilder();
+            AddDefaults(builder);
             configure?.Invoke(builder);
             return builder.Build();
         }
 
+        public static RazorEngine CreateEmpty(Action<IRazorEngineBuilder> configure)
+        {
+            var builder = new DefaultRazorEngineBuilder();
+            configure?.Invoke(builder);
+            return builder.Build();
+        }
+
+        internal static void AddDefaults(IRazorEngineBuilder builder)
```

Comment:
Will probably want to eventually fork this into `AddRuntimeDefaults` and `AddDesignTimeDefaults`. I feel like it'd be better to put the fork in "is runtime vs design time" logic at the engine builder level and not in the individual phases/features (that's where it currently is in todays Razor, has been very painful/bug farm).